pkg/controller/traceflow/controller.go

```diff
@@ -410,3 +415,16 @@ func (c *Controller) deallocateTag(name string, tag uint8) {
 		}
 	}
 }
+
+func (c *Controller) validateTraceflow(tf *crdv1alpha1.Traceflow) error {
+	if !tf.Spec.LiveTraffic {
+		srcPod, err := c.podInformer.Lister().Pods(tf.Spec.Source.Namespace).Get(tf.Spec.Source.Pod)
+		if err != nil {
+			return err
```

Comment:
could you check the error type here? If it's `NotFound`, maybe wrap it with your own error message

Reply:
Fixed.